operations/mimir/ruler-query-path.libsonnet

```diff
@@ -0,0 +1,80 @@
+// Deployment of an isolated, dedicated query path for ruler remote evaluation.
+{
+  _config+:: {
+    ruler_query_path_enabled: false,
```

Comment:
Can we remove this, in favour of `ruler_remote_evaluation_enabled`?

Reply:
We need two options to support a clean migration, otherwise we risk rule evaluations failing before the read path is fully operational.

So that the first option is consistent with the filename, how about:

```
ruler_remote_evaluation_enabled: false
ruler_remote_evaluation_query_path_enabled: self.ruler_remote_evaluation_enabled,
```

The second can default to the first, so new installations only need to set enabled.

For clean migrations, the process will be:
1. Set `ruler_remote_evaluation_query_path_enabled: true`
2. Set `ruler_remote_evaluation_enabled: true`